minerl/data/data_pipeline.py

```diff
@@ -133,7 +130,7 @@ def load_data(self, stream_name: str, skip_interval=0, include_metadata=False, i
 
         # make a copty  
         gym_spec = gym.envs.registration.spec(self.environment)
-        target_space = copy.deepcopy(gym_spec._kwargs['observation_space'])
+        target_space = copy.deepcopy(self.observation_space)
```

Comment:
Above we have self._env_spec.observationspace and here we have self.observation_space - are these the same thing?

Reply:
@shwang: Figure out who uses `_kwargs` still. (and report to Brandon)